src/django_picklefield/__init__.py

```diff
@@ -0,0 +1,10 @@
+import django.utils.version
```

Comment:
You can remove the version related contents of this file, it's not needed anymore.